scripts/run-e2e.mjs

```
@@ -1,45 +1,64 @@
-    if (normalizedNextEnvironment !== currentNextEnvironment) {
-      await writeFile(nextEnvironmentPath, normalizedNextEnvironment);
+    // Next dev writes the generated file for its current distDir; leave a
+    // canonical local copy without making it repository state.
+    try {
+      const currentNextEnvironment = await readFile(nextEnvironmentPath, "utf8");
+      const normalizedNextEnvironment = normalizeNextEnvironment(currentNextEnvironment);
+      if (normalizedNextEnvironment !== currentNextEnvironment) {
+        await writeFile(nextEnvironmentPath, normalizedNextEnvironment);
+      }
+    } catch (error) {
+      if (error?.code !== "ENOENT") throw error;
```

Biome (2.5.6)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)